apps/client/public/service-worker.js

```diff
@@ -0,0 +1,25 @@
+self.addEventListener('install', () => {
```

Comment:
this cannot be `.ts`?

Reply:
It can, but we need to transpile it to JS.